plugins/jenkins/src/plugin.ts

```diff
@@ -21,27 +21,29 @@ import {
   createRoutableExtension,
   createRouteRef,
   discoveryApiRef,
+  identityApiRef,
 } from '@backstage/core';
-import { JenkinsApi, jenkinsApiRef } from './api';
+import { JenkinsApiImpl, jenkinsApiRef } from './api';
 
 export const rootRouteRef = createRouteRef({
   path: '',
   title: 'Jenkins',
 });
 
 export const buildRouteRef = createRouteRef({
-  path: 'run/:branch/:buildNumber',
-  params: ['branch', 'buildNumber'],
-  title: 'Jenkins run',
+  path: 'build/:jobName/:buildNumber',
+  params: ['jobName', 'buildNumber'],
+  title: 'Jenkins build',
```

Comment:
This is a fairly significant change that may affect others that depend on that route - the changeset for this package should inform users about that and mention some brief migration instructions so they can read about it in the changelog

Reply:
I have added a changeset description of the change. If people think it is too big a change, I can investigate keeping the old route (deprecated). It won't be trivial but isn't impossible either.